examples/practice-typescript/src/components/Logo/logo.scss

```diff
@@ -0,0 +1,18 @@
+.icon-logo {
+  background-image: url('../../assets/images/managers/logo-manager.svg');
```

Comment:
Why use background image? We can use svg component. And also svg should including the company name not separate
https://blog.logrocket.com/how-to-use-svgs-react/

Reply:
Fixed